homeassistant/components/starline/sensor.py

```diff
@@ -70,6 +70,12 @@
         device_class=SensorDeviceClass.DISTANCE,
         icon="mdi:counter",
     ),
+    SensorEntityDescription(
+        key="gps_count",
+        translation_key="gps_count",
+        icon="mdi:satellite-variant",
+        native_unit_of_measurement="",
```

Comment:
I think you can just omit the unit specifier if there are no units.

Reply:
Oh lol, apparently I have a draft review comment saying it should be "satellites"

Reply:
![image](https://github.com/home-assistant/core/assets/1973755/f847074d-1fac-4b9a-a21e-c36ee0e46390)
Looks strange.

Reply:
I looked at the GitHub integration and it specifies custom units for each type of counting sensor (Issues, Forks, Stars, etc).

What looks strange?

Reply:
GPS Satellites: 9 satellites
Strange to duplicate "satellites" here

Reply:
I mean usually we exclude the UoM from the entity name, but in this case there is no real alternative (Like "total kWh" would become "total energy", but in this case removing satellites leaves us with an entity without a lot of context).

I think there is no better alternative